src/ast.cr

```diff
@@ -111,6 +115,18 @@ module Mint
             )
           end
 
+      @unified_locales =
+        @locales
+          .group_by(&.language)
+          .map do |_, locales|
+            Locale.new(
+              input: Data.new(input: "", file: ""),
+              fields: locales.flat_map(&.fields),
```

Comment:
I'm guessing whichever `locale` is parsed last is the one that's used? (for the same language)

Reply:
No, it's merging all the keys from all the instances.